docs/content/references/cli/name.mdx

```diff
@@ -0,0 +1,329 @@
+If you try to burn a name that is not expired, you will receive an error message indicating that the name is still active.
+
+```bash
+iota name burn awesome.iota
+NFT for awesome.iota has not expired yet: 2026-06-03 17:53:01.182000000 UTC
+```
+
+### List
+
+The `iota name list` command displays all domains currently owned by the active wallet address (or a specified address).
+
+```bash
+iota name list
+╭────────────────────────────────────────────────────────────────────┬──────────────┬───────────────────────────────────────────────────╮
+│ id                                                                 │ domain       │ expiration                                        │
+├────────────────────────────────────────────────────────────────────┼──────────────┼───────────────────────────────────────────────────┤
+│ 0x831b7d983d20914950fe98ca49ffcd366b55fca593ede95805869c9a090a8de8 │ abc.iota     │ 1780512084730 (2026-06-03 18:41:24.730000000 UTC) │
+│ 0x896cee37d691bf053e5883fa3f696be90329d67adf879b1323f01982d9a00466 │ awesome.iota │ 1780509181182 (2026-06-03 17:53:01.182000000 UTC) │
+╰────────────────────────────────────────────────────────────────────┴──────────────┴───────────────────────────────────────────────────╯
+```
+
+
+### Renew
+
+The `iota name renew` command extends the expiration of an existing `.iota` name. The renewal cost is calculated as the domain's base price multiplied by the number of years.
+
+```bash
+iota name renew abc.iota 2
+Renewed record:
+╭────────────────┬────────────────────────────────────────────────────────────────────╮
+│ field          │ value                                                              │
+├────────────────┼────────────────────────────────────────────────────────────────────┤
+│ NFT ID         │ 0x831b7d983d20914950fe98ca49ffcd366b55fca593ede95805869c9a090a8de8 │
+│ Target Address │ 0x55771a075822ffe446b638d64f5b69641992c556471f3c0055fd9646e0c4c3f6 │
+│ Expiration     │ 1843584084730 (2028-06-02 18:41:24.730000000 UTC)                  │
+╰────────────────┴────────────────────────────────────────────────────────────────────╯
+Updated NFT:
+╭────────────┬────────────────────────────────────────────────────────────────────╮
+│ field      │ value                                                              │
+├────────────┼────────────────────────────────────────────────────────────────────┤
+│ ID         │ 0x831b7d983d20914950fe98ca49ffcd366b55fca593ede95805869c9a090a8de8 │
+│ Domain     │ abc.iota                                                           │
+│ Expiration │ 1843584084730 (2028-06-02 18:41:24.730000000 UTC)                  │
+╰────────────┴────────────────────────────────────────────────────────────────────╯
+Transaction digest: 5F2B4saqVPzDExdnCm3p9goCLDx3BCApt3zpDfvWz9Lq
+```
+
+
+### Transfer
+
+The `iota name transfer` command transfers a domain to a different address by transferring the NFT associated with the domain.
```

Comment:
Probably good to mention here, that the owner of the `to_address` should set the target address to `to_address`, because otherwise he'll own the name, but it potentially still links to another address (or to no address at all)